modular_chomp/code/modules/clothing/glasses/glasses.dm

```diff
@@ -6,3 +6,28 @@
 	icon_state = "AR_visor_tesh"
 	item_state = "AR_visor_tesh"
 	sprite_sheets = list(SPECIES_TESHARI = 'icons/inventory/eyes/mob_teshari.dmi')
+
+
+/obj/item/clothing/glasses/darksight
+	name = "voidviewer goggles"
+	desc = "Cryptic goggles designed for seeing beyond the viel of normality."
+
+	icon = 'modular_chomp/icons/inventory/eyes/glasses.dmi'
+	icon_override = 'modular_chomp/icons/inventory/eyes/mob.dmi'
+	icon_state = "darksight"
+
+	action_button_name = "Toggle Goggles"
+
+	toggleable = 1
+	enables_planes = list(INVISIBILITY_SHADEKIN)
+
+/obj/item/clothing/glasses/meson/darksight
+	name = "optical meson scanner"
+	desc = "Used for seeing walls, floors, and stuff through anything."
+	icon_state = "meson"
+	item_state_slots = list(slot_r_hand_str = "meson", slot_l_hand_str = "meson")
+	action_button_name = "Toggle Goggles"
+	origin_tech = list(TECH_MAGNET = 2, TECH_ENGINEERING = 2)
+	toggleable = 1
+	vision_flags = SEE_TURFS
+	enables_planes = list(VIS_FULLBRIGHT, VIS_MESONS, INVISIBILITY_SHADEKIN)
```

Comment:
enables_planes = list(VIS_FULLBRIGHT, VIS_MESONS, ~~INVISIBILITY_SHADEKIN~~)
see_invisible = INVISIBILITY_SHADEKIN

Reply:
if making it consistent, SEE_INVISIBILITY_SHADEKIN

create a new define in misc.dm with that name, value 55